releases/note-source/v17_0.rst

```diff
@@ -0,0 +1,299 @@
+
+__ https://community.lsst.org/t/sourcecatalog-flux-fields-are-now-instflux/3265
+
+.. _release-v17-0-inverted:
+
+Use ``inverted`` to  get an inverse transform
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, the objects in the codebase representing coordinate transforms (e.g. `lsst.geom.LinearTransform`, `astshim.Mapping`) supported a heterogeneous mix of APIs for retrieving the inverse transform.
+These have now all been standardized to use the method ``inverted()``.
+For more information, refer to `this community.lsst.org post`__ and :jira:`RFC-500`.
+
+__ https://community.lsst.org/t/inverted-is-now-the-standard-method-name-to-get-an-inverse-transform/3122
+
+.. _release-v17-0-calibration-flags:
+
+Changes to calibration flags
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following changes were made to table field names describing sources used for calibration:
+
+- ``calib_psfCandidate`` was renamed ``calib_psf_candidate``;
+- ``calib_psfUsed`` was renamed ``calib_psf_used``;
+- ``calib_astrometryUsed`` was renamed ``calib_astrometry_used``.
+
+For more information, refer to `this community.lsst.org post`__ and :jira:`RFC-498`.
+
+__ https://community.lsst.org/t/change-in-calibration-flag-names-as-per-rfc-498/3170
+
+.. _release-v17-0-blendedness-flags:
+
+Changes to blendedness flags
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following changes were made to table field names describing source blendedness:
+
+- ``base_Blendedness_raw_instFlux`` was renamed to ``base_Blendedness_raw``;
+- ``base_Blendedness_abs_instFlux`` was renamed to ``base_Blendedness_abs``;
+- ``base_Blendedness_raw_instFlux_child`` was renamed to ``base_Blendedness_raw_child_instFlux``;
+- ``base_Blendedness_raw_instFlux_parent`` was renamed to ``base_Blendedness_raw_parent_instFlux``;
+- ``base_Blendedness_abs_instFlux_child`` was renamed to ``base_Blendedness_abs_child_instFlux``;
+- ``base_Blendedness_abs_instFlux_parent`` was renamed to ``base_Blendedness_abs_parent_instFlux``;
+- ``deblend_psfFlux`` was renamed to ``deblend_psf_instFlux``.
+
+For more information, refer top `this community.lsst.org post`__ and :jira:`RFC-534`.
+
+__ https://community.lsst.org/t/change-in-naming-of-base-blendedness-and-meas-deblender-fields/3351
+
+.. _release-v17-0-task-metadata:
+
+Task metadata is now written as YAML
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, metadata was stored as a mix of YAML__ and Boost.serialization__; now, only the former is used.
+For more information, refer to `this community.lsst.org post`__.
+
+__ https://yaml.org
+__ https://www.boost.org/doc/libs/1_69_0/libs/serialization/doc/index.html
+__ https://community.lsst.org/t/task-metadata-now-written-as-yaml/3192
+
+.. _release-v17-0-pybind11:
+
+Changes to wrapping C++ code into Python
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+DM uses pybind11__ to expose functionality written in C++ to the Python level.
```

Comment:
Doesn't pybind11 need back ticks to make the URL linking work?

Reply:
The link works for me